flow/scripts/global_route.tcl

```diff
@@ -24,18 +24,20 @@ if {[info exist env(FASTROUTE_TCL)]} {
 global_route -guide_file $env(RESULTS_DIR)/route.guide \
                -congestion_report_file $env(REPORTS_DIR)/congestion.rpt \
                {*}[expr {[info exists ::env(GLOBAL_ROUTE_ARGS)] ? $::env(GLOBAL_ROUTE_ARGS) : \
-               {-congestion_iterations 30 -congestion_report_iter_step 5 -verbose}}]
+               {-congestion_iterations 50 -congestion_report_iter_step 5 -verbose}}]
 
 
 set_placement_padding -global \
     -left $::env(CELL_PAD_IN_SITES_DETAIL_PLACEMENT) \
     -right $::env(CELL_PAD_IN_SITES_DETAIL_PLACEMENT)
-repair_antennas -iterations 5
-check_placement -verbose
 
 set_propagated_clock [all_clocks]
 estimate_parasitics -global_routing
 
+if {[info exist env(DONT_USE_CELLS)]} {
+  set_dont_use $::env(DONT_USE_CELLS)
+}
+
 if { [info exists ::env(RECOVER_POWER)] } {
```

Comment:
recover_power should happen after incremental repair

Reply:
Fixed